### PR TITLE
[qt] Fixed freezing UI during search.

### PR DESCRIPTION
**Pull request by ygorshenin**
@mpimenov @syershov PTAL

### CODE REVIEW

File: qt/search_panel.hpp [hunk 1]
```diff
@@ -3,6 +3,9 @@
 #include "search/result.hpp"
 #include "search/everywhere_search_params.hpp"
 
+#include "base/thread_checker.hpp"
+
+#include "std/cstdint.hpp"
```
Comment (syershov) [inline qt/search_panel.hpp]: #include <cstdint>

Comment (syershov) [inline qt/search_panel.hpp]: And so on...

Comment (ygorshenin) [inline qt/search_panel.hpp]: Done

**Comment by ygorshenin** [comment]
@syershov @mpimenov PTAL

**Pull request merged**